src/coreclr/jit/emitfmtswasm.h

```diff
@@ -31,6 +31,12 @@ IF_DEF(OPCODE,  IS_NONE, NONE) // <opcode>
 IF_DEF(BLOCK,   IS_NONE, NONE) // <opcode> <0x40>
 IF_DEF(LABEL,   IS_NONE, NONE) // <ULEB128 immediate>
 IF_DEF(ULEB128, IS_NONE, NONE) // <opcode> <ULEB128 immediate>
+IF_DEF(LEB128,  IS_NONE, NONE) // <opcode> <LEB128 immediate (signed)>
```

Comment:
Can we do `SLEB` for signed, and in other places where applicable?
```suggestion
IF_DEF(SLEB128,  IS_NONE, NONE) // <opcode> <SLEB128 immediate>
```